Composer/packages/client/src/components/ProjectTree/ProjectTree.tsx

```diff
@@ -358,6 +366,20 @@ export const ProjectTree: React.FC<Props> = ({
           onDialogCreateTrigger(skillId, dialog.id);
         },
       },
+      {
+        label: formatMessage('Create KB from scratch'),
+        onClick: () => {
+          createQnAFromScratchDialogBegin({ projectId: skillId });
+          onDialogCreateQnAKBFromUrl(skillId, dialog.id);
```

Comment:
I think you have these callbacks in reverse with line 380

Reply:
Addressed it in the PR now